src/Login.ts

```diff
@@ -179,26 +185,30 @@ export default class Login {
     }
 }
 
+/**
+ * Describes the OIDC native login flow
+ * Separate from js-sdk's `LoginFlow` as this does not use the same /login flow
+ * to which that type belongs.
+ */
 export interface OidcNativeFlow extends ILoginFlow {
     type: "oidcNativeFlow";
+    // this client's id as registered with the configured OIDC OP
     clientId: string;
 }
 /**
  * Finds static clientId for configured issuer, or attempts dynamic registration with the OP
+ * Returns OIDC native flow when client is ready to attempt login via OIDC native flow
  * @param delegatedAuthConfig  Auth config from ValidatedServerConfig
```

Comment:
I think there should be a blank line here too. And this one probably *could* be enforced somehow, but I'm afraid I don't have the time to go and figure out how.

```suggestion
 *
 * @param delegatedAuthConfig  Auth config from ValidatedServerConfig
```